swagger/v1-prerelease/swagger.yaml

```diff
@@ -2052,6 +2052,57 @@ paths:
             application/json:
               schema:
                 type: string
+  /_schema/versions:
```

Comment:
SchemaController comes from healthcare-Shared component and does not have versions set. So it was using the config to use default which is V1. This is a good thing now we are explicitly putting a version on it and can deprecate v1

Reply:
Looks like it's still versionless in v2, like health/check.

Reply:
Checked with @brandonpollett, we should be able to remove this now - perhaps we should do this as part of v2

Reply:
It will have V2. I have not generated the swagger for V2 because it is behind FF. I have Feature 100502 to track all the work to make V2 public